packages/docs/components/text-input.md

```diff
@@ -355,11 +355,182 @@ Finally, Users with low digital literacy may not understand the purpose or behav
 :::
 
 ::: slot html-scss
-## HTML & CSS
 
-### Invalid
 
-The `:invalid` CSS pseudo-class represents any `<input>` or other `<form>` element whose contents fail to validate. - MDN
-Because of the current inability to ensure consistent validation behavior across browsers, we're using the `[data-invalid]` attribute to indicate this state.
+## Types
```

Comment:
We're missing examples for Textarea and the two search variants

Reply:
Added